--- a/src/OpenColorIO/CMakeLists.txt
+++ b/src/OpenColorIO/CMakeLists.txt
@@ -28,31 +28,6 @@ set(SOURCES
     Context.cpp
     ContextVariableUtils.cpp
     CPUInfo.cpp
-    fileformats/cdl/CDLParser.cpp
-    fileformats/cdl/CDLReaderHelper.cpp
-    fileformats/cdl/CDLWriter.cpp
-    fileformats/ctf/CTFReaderHelper.cpp
-    fileformats/ctf/CTFReaderUtils.cpp
-    fileformats/ctf/CTFTransform.cpp
-    fileformats/ctf/IndexMapping.cpp
-    fileformats/FileFormat3DL.cpp
-    fileformats/FileFormatCCC.cpp
-    fileformats/FileFormatCC.cpp
-    fileformats/FileFormatCDL.cpp
-    fileformats/FileFormatCSP.cpp
-    fileformats/FileFormatCTF.cpp
-    fileformats/FileFormatDiscreet1DL.cpp
-    fileformats/FileFormatHDL.cpp
-    fileformats/FileFormatICC.cpp
-    fileformats/FileFormatIridasCube.cpp
-    fileformats/FileFormatIridasItx.cpp
-    fileformats/FileFormatIridasLook.cpp
-    fileformats/FileFormatPandora.cpp
-    fileformats/FileFormatResolveCube.cpp
-    fileformats/FileFormatSpi1D.cpp
-    fileformats/FileFormatSpi3D.cpp
-    fileformats/FileFormatSpiMtx.cpp
-    fileformats/FileFormatTruelight.cpp
     fileformats/FileFormatUtils.cpp
-    fileformats/FileFormatVF.cpp
     fileformats/FormatMetadata.cpp
-    fileformats/xmlutils/XMLReaderHelper.cpp
-    fileformats/xmlutils/XMLReaderUtils.cpp
-    fileformats/xmlutils/XMLWriterUtils.cpp
     CPUProcessor.cpp
